Misha/Files/mp3_tagger/tagger_main.py

```diff
@@ -0,0 +1,45 @@
+import eyed3
+
+
+def input_mp3_file_name(mp3_file_name):
+    file_split = str(mp3_file_name).split('.')
+    if len(file_split) == 1 or file_split[len(file_split)-1] != 'mp3':
+        raise TypeError('Invalid file type')
+    else:
+        try:
+            audiofile = eyed3.load(mp3_file_name)
+            return audiofile
+        except OSError:
+            raise OSError('file not found')
+
+
+def get_tag(audiofile):
+    tags_dict = {'artist': audiofile.tag.artist,
+                 'album': audiofile.tag.album,
+                 'album artist': audiofile.tag.album_artist,
+                 'title': audiofile.tag.title,
+                 'track num': audiofile.tag.track_num}
+    return tags_dict
+
+
+def set_tag(audiofile, tags_dict):
```

Comment:
Please remove one line function

Reply:
So if i remove? how should i change tag? All this information (album artist etc.) is in TAG, so if i want to change TAG i should have tuple-list-dict not one value.